src/css/_buttons.scss

```diff
@@ -21,4 +21,49 @@
     color: var(--color-primary);
     border-color: var(--color-primary);
   }
+
+  &.il-white-blue {
+    color: var(--il-blue);
```

Comment:
@bryanjonker-illinois Let's try to utiliize the --color-primary & --color-secondary variables for these. So this one could be:
```
&.il-white-blue {
  --color-primary: varI--il-blue);
}
```